providers/openshift/provider.go

```diff
@@ -310,6 +331,21 @@ func (p *OpenShiftProvider) Complete(data *providers.ProviderData, reviewURL *ur
 	p.ProviderData = data
 	p.ReviewURL = reviewURL
 
+	kubeClient, err := p.KubeClientOptions.ToKubeClientConfig()
+	if err != nil {
+		return err
+	}
+
+	kubeInformersMachineNS := informers.NewSharedInformerFactoryWithOptions(
```

Comment:
what is the meaning of `Machine`?

Reply:
should've probably been `MachineConfig`. It's the NS with the configs that are maintained by operators, hence the "machine"

Reply:
:robot: